docs/docsite/rst/user_guide/playbooks_variables.rst

```diff
@@ -308,25 +308,30 @@ Understanding variable precedence
 Ansible does apply variable precedence, and you might have a use for it. Here is the order of precedence from least to greatest (the last listed variables override all other variables):
 
   #. command line values (for example, ``-u my_user``, these are not variables)
-  #. role defaults (defined in role/defaults/main.yml) [1]_
+  #. role defaults from all roles (defined in ``roles/*/defaults/main.yml``)
+  #. role dependency defaults for tasks from a role (defined in ``roles/{dependent_roles}/defaults/main.yml``)
+  #. role defaults for tasks from a role (defined in ``role/defaults/main.yml``) [1]_ [8]_ [9]_
   #. inventory file or script group vars [2]_
-  #. inventory group_vars/all [3]_
-  #. playbook group_vars/all [3]_
-  #. inventory group_vars/* [3]_
-  #. playbook group_vars/* [3]_
+  #. inventory ``group_vars/all`` [3]_
+  #. playbook ``group_vars/all`` [3]_
+  #. inventory ``group_vars/*`` [3]_
+  #. playbook ``group_vars/*`` [3]_
   #. inventory file or script host vars [2]_
-  #. inventory host_vars/* [3]_
-  #. playbook host_vars/* [3]_
-  #. host facts / cached set_facts [4]_
-  #. play vars
-  #. play vars_prompt
-  #. play vars_files
-  #. role vars (defined in role/vars/main.yml)
-  #. block vars (only for tasks in block)
-  #. task vars (only for the task)
-  #. include_vars
-  #. set_facts / registered vars
-  #. role (and include_role) params
+  #. inventory ``host_vars/*`` [3]_
+  #. playbook ``host_vars/*`` [3]_
+  #. host facts / cached ``set_facts`` [4]_
+  #. play ``vars``
+  #. play ``vars_prompt``
+  #. play ``vars_files``
+  #. role vars from all roles (defined in ``roles/*/vars/main.yml`` or ``vars:``)
+  #. role dependency vars for tasks from a role (defined in ``roles/{dependent_roles}/vars/main.yml``)
+  #. role vars for tasks from a role (defined in ``role/vars/main.yml``) [5]_ [8]_ [9]_
+  #. block ``vars`` (only for tasks in block)
+  #. task ``vars`` (only for the task)
+  #. ``include_vars``
+  #. ``set_facts`` / registered vars
+  #. role dependency params for tasks from a role
+  #. role params for tasks from a role [6]_ [7]_
   #. include params
```

Comment:
Try to indicate that these are task include params, and not role includes, since role includes do not support params.